jrcc-access-spring-boot-sample-app/src/main/resources/application.yml

```diff
@@ -7,13 +7,14 @@ logging:
         bc: DEBUG
   pattern:
     console: "%d{yyyy-MM-dd HH:mm:ss} [%thread] %-5level %logger{36} %X{transaction.filename} %X{transaction.id} - %msg%n"
-    file: "%d{yyyy-MM-dd HH:mm:ss} [%thread] %-5level %logger{36} %X{transaction.filename} %X{transaction.id} - %msg%n"
+    file: "%d{yyyy-MM-dd HH:mm:ss} [%thread] %-5level %logger{36} %X{transaction.filename} - %msg%n"
 bcgov:
   access:
+    #####input settings for sftp
```

Comment:
do we need this here?

Reply:
:) bad me.

Reply:
not bad at all, you should see what I leave in the code before it's merged, it just mean our level of details is going up ;-)